### PR TITLE
Restore timero photo

### PR DESCRIPTION
**Pull request by NerdyChara**
# Pull Request

## Description

Updates Timero's link to use the current DIscord url. Eventually I want to ensure that all images are from the app itself but for now it's solid. 
## Related Issue

Closes #XXXXX

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -204,7 +204,7 @@ <h4>Community Manager</h4>
           </div>
           <div onclick="modcard('_template')" class="card">
             <img
-              src="https://images-ext-2.discordapp.net/external/-5NZHIaVUESbwV0Qb4CaxpP6SfvZz408sWYDdM8_0A0/https/cdn.discordapp.com/avatars/342472606257250314/c0d17fe297a474b8063ac5a57289b7c9.webp"
+              src="https://cdn.discordapp.com/avatars/342472606257250314/239cbf0523a88afecce5f4ce6e3c8dd2.webp"
```
Comment (naomi-lgbt) [inline index.html:207]: Since we are using the Discord link, we don't need to upload the photo in this PR.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Looks good, apart from what Naomi said. Thanks. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: index.html (hunk 1 above)
Comment (NerdyChara) [reply index.html:207]: I thought I deleted it on my branch. Let me go check it. ._.

Comment (NerdyChara) [reply index.html:207]: Okay there it is. It's gone now.

**Review by naomi-lgbt — APPROVED** [review]
Awesome let's get this in.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]
Amazing! Thanks. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Pull request merged**